com.musala.simple.students.spring/src/main/java/com/musala/simple/students/spring/StudentsController.java

```diff
@@ -0,0 +1,72 @@
+package com.musala.simple.students.spring;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.Valid;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.view.RedirectView;
+
+import com.musala.simple.students.spring.database.DatabaseType;
+import com.musala.simple.students.spring.student.Student;
+
+@Controller
+public class StudentsController {
+    
+    @PostMapping("students/add")
+    public ResponseEntity<?> addStudentAjax(@Valid @RequestBody Student student, Errors errors) {
+        StudentService.addStudent(student, DatabaseType.MongoDb);
+
+        AjaxResponseBody result = new AjaxResponseBody();
+        if (errors.hasErrors()) {
+            result.setMsg(errors.getAllErrors()
+                        .stream().map(x -> x.getDefaultMessage())
+                        .collect(Collectors.joining(",")));
+            return ResponseEntity.badRequest().body(result);
+        }
+        result.setMsg("success");
+        result.setResult(null);
+
+        return ResponseEntity.ok(result);
+    }
+
+    @RequestMapping(value = "students/all")
+    public @ResponseBody String showAllStudents(){
+        List<Student>studentsList = StudentService.getStudents(DatabaseType.MongoDb);
+        StringBuilder sBuilder = new StringBuilder();
+        for (Student student : studentsList) {
+            sBuilder.append("<div>")
```

Comment:
As discussed, when Angular is applied this method should and will return only the data returned from the database, so I will leave this html for now as it will be removed very soon.